packages/expo-av/android/CMakeLists.txt

```diff
@@ -0,0 +1,60 @@
+cmake_minimum_required(VERSION 3.4.1)
+
+set (CMAKE_VERBOSE_MAKEFILE ON)
+set (CMAKE_CXX_STANDARD 14)
+
+set (PACKAGE_NAME "expo-av")
+set (BUILD_DIR ${CMAKE_SOURCE_DIR}/build)
+
+# shared
+
+add_library(
+        ${PACKAGE_NAME}
+        SHARED
+        src/main/cpp/EXAV.cpp
+        src/main/cpp/JPlayerData.cpp
+        src/main/cpp/JAVManager.cpp
+        "${REACT_NATIVE_DIR}/ReactCommon/jsi/jsi/jsi.cpp"
```

Comment:
in RN 0.67, jsi is a shared library you can link with. it's not necessary to build jsi.cpp here.